server/cumulative-trend-service/package.json

```diff
@@ -5,13 +5,15 @@
   "main": "server.js",
   "scripts": {
     "start": "if-env NODE_ENV=production && npm run start:prod || npm run start:dev",
-    "start:prod": "node server.js",
-    "start:dev": "nodemon"
+    "start:prod": "nodemon server.js",
```

Comment:
for prod, it needs to be "node server.js"

Reply:
OK